Sources/Bond/AppKit/NSCollectionView+DataSource.swift

```diff
@@ -82,7 +82,12 @@ open class CollectionViewBinderDataSource<Changeset: SectionedDataSourceChangese
     open func applyChangeset(_ changeset: Changeset) {
         guard let collectionView = collectionView else { return }
         let diff = changeset.diff.asOrderedCollectionDiff.map { $0.asSectionDataIndexPath }
-        if diff.isEmpty {
+
+        guard !diff.isEmpty else {
```

Comment:
Are there similar changes necessary for the other AppKit controls like `NSTableView`?

Reply:
NSTableView uses patch instead of diff (not sure for which reason). I don't code for macOS at all and don't know if my problem exists there.

I'm not sure if my changes are necessary there, but for safety purpose, it's better not to apply them

Reply:
I'd probably apply them to `NSTableView` under the assumption that `reloadData` is going to be just as expensive there, but I'm happy to take that up after this is reviewed and merged 👍